p2p/address-book/src/lib.rs

```diff
@@ -29,12 +31,24 @@ pub struct AddressBookConfig {
     ///
     /// Gray peers are peers we are yet to make a connection to.
     pub max_gray_list_length: usize,
-    /// The location to store the address book.
-    pub peer_store_file: PathBuf,
+    /// The location to store the peer store file.
+    pub peer_store_folder: PathBuf,
     /// The amount of time between saving the address book to disk.
     pub peer_save_period: Duration,
 }
 
+#[cfg(feature = "serde_config")]
+impl Default for AddressBookConfig {
+    fn default() -> Self {
+        Self {
+            max_white_list_length: 1000,
+            max_gray_list_length: 5000,
+            peer_store_folder: cuprate_helper::fs::CUPRATE_CACHE_DIR.clone(),
+            peer_save_period: Duration::from_secs(90),
```

Comment:
if its the default is there any corresponding constant in `cuprate-constants` ?

Reply:
Nope, should I? there are a lot of constant values that I just define in the default method as that's the only place they are used, defining them elsewhere is more annoying IMO